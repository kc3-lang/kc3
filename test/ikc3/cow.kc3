quote cow 1
cow 1
quote a = cow 1
a = cow 1
quote a <- 2
a <- 2
quote a
a
