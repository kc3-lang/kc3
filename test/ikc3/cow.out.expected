cow 1
cow 1
a = cow 1
cow 1
a <- 2
cow 2
a
cow 2
